test/application_system_test_case.rb

```diff
@@ -0,0 +1,5 @@
+require "test_helper"
```

Comment:
can you please remove the whole test folder. as we will be using rspec we don't need to keep this test folder.

Reply:
Removed bhai. Thank you

Reply:
Please address this review ⬆️

Reply:
Sorry for the inconvenience apu and bhai. Fixed this now. Thank you